decidim-participatory_processes/config/locales/en.yml

```diff
@@ -118,6 +122,14 @@ en:
         update:
           error: There was en error updating this participatory process group.
           success: Participatory process group updated successfully.
+      participatory_process_private_users:
+        destroy:
+          success: Participatory process private user destroyed successfully.
```

Comment:
I think this naming might be confusing for the end user. What about something like: "Private Process User Access"? Would this be clearer?

In case of doubt refer to @decidim/product 🤷‍♂️